features/gezag-minderjarige/1.2-is-persoon-minderjarig-en-niet-overleden.feature

```diff
@@ -0,0 +1,111 @@
+
+  Voor inwerkingtreding van de Wet Tegengaan Huwelijksdwang op 5-12-2015 kon iemand meerderjarig worden door afsluiten van een huwelijk of
+  geregistreerd partnerschap. Dit is bijna 5 jaar geleden. Iemand die nu 17 jaar is zou dan op 12 jarige leeftijd of eerder getrouwd moeten zijn, wil dit
+  nog van toepassing zijn hier wordt daarom geen rekening mee gehouden.
+
+  Hoe te achterhalen:
+  De leeftijd van de persoon moet 17 jaar of minder zijn en de persoon moet niet zijn overleden.
+
+  Aanbeveling
+  Er is nog geen oplossing voor de meerderjarigheidsverklaring. Het idee is dat persoon de letter M krijgt bij indicatie
+  gezag (11.32.10). Zodra dit het geval is moet deze informatie worden meegenomen in de beoordeling en bij aanwezigheid van deze indicatie
+  wordt de persoon behandeld als meerderjarige.
+
+  Gebruikte velden:
+  - datum opschorting bijhouding (67.10)
+  - geboortedatum (03.10)
+
+  Achtergrond:
+    Gegeven de persoon 'Laura' met burgerservicenummer '000000012'
+    * is meerderjarig
+    En de persoon 'Ludo' met burgerservicenummer '000000024'
+    * is meerderjarig
+    En 'Laura' en 'Ludo' zijn met elkaar gehuwd
+    En de persoon 'Linda' met burgerservicenummer '000000036'
+    * is ingeschreven in de BRP
+    * is niet geëmigreerd geweest
+    * is in Nederland geboren
+    * heeft 'Laura' als ouder 1
+    * heeft 'Ludo' als ouder 2
+
+  Regel: gezag wordt niet bepaald wanneer het kind overleden is
+
+    Scenario: Wanneer gezag wordt opgevraagd van een overleden kind, wordt geen gezag teruggeven.
+      Gegeven persoon 'Linda'
+      * is minderjarig
+      En bijhouding van de persoonsgegevens van 'Linda' is opgeschort met de volgende gegevens
+      | datum opschorting bijhouding (67.10) |
+      | gisteren                             |
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon geen gezag
+
+  Regel: gezag wordt niet bepaald wanneer het kind meerderjarig is
+
+    Scenario: Wanneer gezag wordt opgevraagd van een kind die meerderjarig is, wordt geen gezag teruggeven.
+      Gegeven persoon 'Linda'
+      * is meerderjarig
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon geen gezag
+
+  Regel: gezag wordt niet bepaald wanneer leeftijd onbekend is
+
+    Scenario: Wanneer gezag wordt opgevraagd van een kind waarvan leeftijd onbekend is, wordt geen gezag teruggeven.
+      Gegeven heeft de volgende gegevens
```

Comment:
Deze gegeven stap heeft nu ook wat implementatie detail, dat onbekend geboorte datum 00000000 is.
Misschien hier een gegevenstap van maken: `* heeft een onbekende geboortedatum`

Reply:
In verband met vakantie sluit ik deze conversatie voortijdig.